tests/integration/test_kfp_functional.py

```diff
@@ -100,9 +107,9 @@ async def test_create_and_monitor_run(kfp_client, create_and_clean_experiment):
     # Create a run from a pipeline file (SAMPLE_PIPELINE) and an experiment (create_experiment).
     # This call uses the 'default' kubeflow service account to be able to edit Workflows
     create_run_response = kfp_client.create_run_from_pipeline_package(
-        pipeline_file=SAMPLE_PIPELINE,
+        pipeline_file=SAMPLE_PIPELINE[kfp_version],
         arguments={},
-        run_name="test-run-1",
+        run_name=f"test-run-{kfp_version}",
```

Comment:
not a requirement, but I think improves clarity
```suggestion
        run_name=f"test-run-sdk-{kfp_version}",
```